customer/service_layer/customer.py

```diff
@@ -17,4 +18,8 @@ def create_customer(
         last_name: str,
         email: str,
     ):
-        pass
+        customer_create = CustomerCreate(
```

Comment:
This is what we'd call the orchestration layer. Basically, pulling together a few different pieces of logic under one interface so that you can easily call it. All that any code that calls this functions needs to know is that calling it creates a customer, not how that works.

Separating this out makes it easy to extent to new services (e.g. write the customer to a second database for redundancy). 

At this level we might think about transactions. What happens if, for example, the write to the database fails? Do we want to push on and try to send via the event service, or stop? Similarly if the event service returns an error should we rollback the write to the database? Concerns like this are best placed here where we're pulling those separate operations together.